ceph/ceph_admin/jinja_templates/snmp_destination.jinja

```diff
@@ -1,3 +1,9 @@
+{% if "snmp_v3_auth_username" in spec["spec"]["credentials"] %}
+  createuser -e 0x{{ spec["spec"]["engine_id"] }} {{ spec["spec"]["credentials"]["snmp_v3_auth_username"] }} SHA {{ spec["spec"]["credentials"]["snmp_v3_auth_password"] }}
+  authuser log,execute {{ spec["spec"]["credentials"]["snmp_v3_auth_username"] }}
+{% else %}
```

Comment:
Add check for v2 and else command for exception

Reply:
Check for v2 implemented.
Tried adding exception block in jinja file,but every time it was printing the exception block instead of taking it as format for jinja